index.html

```diff
@@ -44,7 +44,149 @@ <h5> I can help you build a product ,
+           <h3>Profesional Art <br>
+                Printing Data</h3>
+           <p>A daily selection of privately 
+               personalized reads; no accounts 
+               or sign-ups required. has been 
+               the industry's standard</p>
+           <div class="language container"> 
+               <span class="html">html</span> 
+               <span class="bootstrap boot1">bootstrap</span>
+               <span class="ruby ">ruby</span>
+               </div>
+           <button type="button" class="see-project"> See Project</button>
+         </div>
+         <div class="profesional p-4">
+           <h3>Profesional Art <br>
+                Printing Data</h3>
+           <p>A daily selection of privately 
+               personalized reads; no accounts 
+               or sign-ups required. has been 
+               the industry's standard</p>
+           <div class="language container"> 
+               <span class="html">html</span> 
+               <span class="bootstrap boot1">bootstrap</span>
+               <span class="ruby ">ruby</span>
+               </div>
+           <button type="button" class="see-project"> See Project</button>
+         </div>
+         <div class="profesional p-5">
+           <h3>Profesional Art <br>
+                Printing Data</h3>
+           <p>A daily selection of privately 
+               personalized reads; no accounts 
+               or sign-ups required. has been 
+               the industry's standard</p>
+           <div class="language container"> 
+               <span class="html">html</span> 
+               <span class="bootstrap boot1">bootstrap</span>
+               <span class="ruby ">ruby</span>
+               </div>
+           <button type="button" class="see-project"> See Project</button>
+         </div>
+         <div class="profesional p-6">
+           <h3>Profesional Art <br>
+                Printing Data</h3>
+           <p>A daily selection of privately 
+               personalized reads; no accounts 
+               or sign-ups required. has been 
+               the industry's standard</p>
+           <div class="language container"> 
+               <span class="html">html</span> 
+               <span class="bootstrap boot1">bootstrap</span>
+               <span class="ruby ">ruby</span>
+               </div>
+           <button type="button" class="see-project"> See Project</button>
+         </div>
+   </section>
+   <section class="about">
+       <h1> About me</h1>
+       <p>Hello I’m a software developer!
+            I can help you build a product , feature
+             or website Look through some of my work 
+            and experience! If you like what you
+            see and have a project you need coded, 
+             don’t hestiate to  contact me.</p> 
+       
+             <button type="button" class="get">Get My Resume</button>
+             <img class="about-img" src="images/Group1.png" alt="">
+             <hr>
```

Comment:
- [ ] Everything is fine, but you have a small issue that needs to fix. please make your `hr` tag lighter by providing the color. See the design below for reference
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/193013070-4cc74ec4-047b-437f-abaf-123cb45ad9fc.png)
### your design
![your](https://user-images.githubusercontent.com/81727988/193013138-c80854c1-1771-4ea6-8005-a5393bdba709.png)